syntax = "proto3";

import "akkaserverless/annotations.proto";

package com.example;

message ProductState {
    string product_id = 1 [(akkaserverless.field).entity_key = true];
    string name = 2;
}

/*
message ProductAdded {
    ProductState product = 1;
}
*/